### PR TITLE
{compiler}[GCCcore/10.3.0] LLVM v12.0.0

### PR DESCRIPTION
**Pull request by robert-mijakovic**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/l/LLVM/LLVM-12.0.0-GCCcore-10.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+toolchainopts = {'cstd': 'gnu++11'}
+
+source_urls = ['https://github.com/llvm/llvm-project/releases/download/llvmorg-%(version)s/']
+sources = ['llvm-%(version)s.src.tar.xz']
+checksums = ['49dc47c8697a1a0abd4ee51629a696d7bfe803662f2a7252a3b16fc75f3a8b50']
+
+builddependencies = [
+    ('binutils', '2.36.1'),
+    ('CMake', '3.20.1'),
+    ('Python', '3.9.5'),
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/l/LLVM/LLVM-12.0.0-GCCcore-10.3.0.eb:26]: @robert-mijakovic you can now use the `-bare` Python from https://github.com/easybuilders/easybuild-easyconfigs/pull/13035

Comment (boegel) [reply easybuild/easyconfigs/l/LLVM/LLVM-12.0.0-GCCcore-10.3.0.eb:26]: +1, makes more sense

Comment (robert-mijakovic) [reply easybuild/easyconfigs/l/LLVM/LLVM-12.0.0-GCCcore-10.3.0.eb:26]: BTW, installing it with bare lead to sanity check failure.
```
== FAILED: Installation ended unsuccessfully (build directory: /dev/shm/build/Python/3.9.5/GCCcore-10.3.0-bare): build failed (first 300 chars): Sanity check failed: sanity check command python -c 'import _ssl' exited with code 1 (output: Traceback (most recent call last):
  File "<string>", line 1, in <module>
ModuleNotFoundError: No module named '_ssl'
) (took 8 min 56 sec)
```

I guess I should open a ticket on that.

Comment (robert-mijakovic) [reply easybuild/easyconfigs/l/LLVM/LLVM-12.0.0-GCCcore-10.3.0.eb]: I have opened a ticket on that https://github.com/easybuilders/easybuild-easyblocks/issues/2466

**Comment by boegel** [comment]
@boegelbot please test @ generoso
CORE_CNT=16

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in total)
easybuild-c7.novalocal - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/dfbfe9aea8ff042e2eeab244269d4dbf for a full test report.

**Comment by boegel** [comment]
superseded by #13402 (LLVM v12.0.1), so closing this PR...

**Pull request closed without merging**